src/cache/cache_hs.c

```diff
@@ -9,84 +9,84 @@
 #include "wt_internal.h"
```

Comment:
Since we are renaming the file, I think it also makes sense to move it somewhere else. It's not really related to the cache now. Perhaps we need a new directory `history_store`? `hs` seems a bit terse for that purpose.
I'm also open to other options.

Reply:
Going with history/hs.c.